packages/ibm-products/src/components/CreateFullPage/CreateFullPageStep.tsx

```diff
@@ -68,7 +154,7 @@ export let CreateFullPageStep = forwardRef(
       shouldIncludeStep,
       secondaryLabel,
       title,
-    });
+    } as any);
```

Comment:
Better avoid using any and try to correct the param object by including the missing properties.